webserver/webserver/src/main/java/io/helidon/webserver/NettyWebServer.java

```diff
@@ -420,6 +426,19 @@ public int port(String name) {
         return address instanceof InetSocketAddress ? ((InetSocketAddress) address).getPort() : -1;
     }
 
+    @Override
+    public void updateTls(WebServerTls tls) {
+        updateTls(tls, WebServer.DEFAULT_SOCKET_NAME);
+    }
+
+    @Override
+    public void updateTls(WebServerTls tls, String socketName) {
+        SslContext context = createSslContext(tls.sslContext(), new HashSet<>(tls.enabledTlsProtocols()), tls.clientAuth());
+        Optional.ofNullable(initializers.get(socketName))
```

Comment:
I know you've refactored that code but the use of optional is not needed. A good old if-null-throw-else would be more appropriate.